chronosphere/resource_notification_policy_independent.go

```diff
@@ -138,13 +140,33 @@ func notificationRoutesFromModel(
 			Severity:       sev,
 			Notifiers:      sliceutil.Map(f.NotifierSlugs, tfid.Slug),
 			RepeatInterval: durationFromSecs(f.RepeatIntervalSecs),
+			GroupBy:        notificationRouteFromGroupByModel(f),
 		})
 	}
 	load(warn, m.Warn)
 	load(critical, m.Critical)
 	return out
 }
 
+func notificationRouteToGroupByModel(r intschema.NotificationRoute) *models.NotificationPolicyRoutesGroupBy {
```

Comment:
same as comments on the other field, this doesn't need `NotificationRoute`, it only needs `r.GroupBy`. Update naming + params, and make sure toModel/fromModel are symmetric